docs/pages/performance/system.js

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import NoSsr from '@material-ui/core/NoSsr';
+import Box from '@material-ui/core/Box';
+
+export default function SxPropBoxMaterialUI() {
+  return (
+    <NoSsr defer>
```

Comment:
Should we also add perf pages using `theme-ui` and `@chakra-ui` Boxes?

Reply:
We could, my initial use case was to render the components to have an idea of the slowest methods.